### DISCUSSION TITLE
초보) ZEPETO.Multiplay 의 index.ts파일은 서버에서 유일하게 실행되는 건가요?

### DISCUSSION THREAD

**Discussion by Spice8876** [naverz/zepeto-studio-kor#533] [5. Q&A]
멀티플레이 예제에서 
onJoin, onTick 같은 함수들이 clientStater의 Start나 OnStateChanged와는 다르게 유저 수에 상관없이 서버에서 한번만 실행되는게 맞나요..?

**Comment by minjujuu** [comment]
https://studio.zepeto.me/guides/multiplay-room
위 링크를 참고해 보시면 좋을 것 같습니다!

onJoin(client: SandboxPlayer)
- Client가 Room에 입장할 때 호출됩니다.

onTick(deltaTime: number)
- SandboxOptions에서 설정된 tickInterval마다 반복적으로 호출됩니다.
